src/framework/components/screen/component.js

```diff
@@ -136,12 +138,29 @@ class ScreenComponent extends Component {
         }
     }
 
+    _bindElement(element) {
+        this._elements.push(element);
+    }
+
+    _unbindElement(element) {
+        const index = this._elements.indexOf(element);
```

Comment:
could this_elements be a Set instead of an array to avoid this possibly costly indexOf operation?

Reply:
I would assume, iterations are more common through this set, than modifications of it. So it is a question of reduced performance on add/remove or iteration.

Reply:
Set uses object pointer, so no hashing.

Reply:
for users with pooled elements we could also provide a `screen.removeElements()` function removes all elements without requiring all the `.indexOf` calls.

Reply:
related: https://stackoverflow.com/questions/39007637/javascript-set-vs-array-performance

Reply:
This is for my own reference but according to this https://v8.dev/blog/hash-code hash codes for Objects are just random numbers. So I guess that's how Set's hash any object reference internally..